intoto/attestation/v1alpha1/types.proto

```diff
@@ -16,29 +16,40 @@ syntax = "proto3";
 
 package intoto.attestation.v1alpha1;
 
-option go_package = "github.com/rode/proto/intoto/attestation/v1alpha1";
-
-// TypeURI is a uniform resource locator that should resolve to a human-readable description of the type to which it refers.
-// See the in-toto attestation description for more information: https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI
-message TypeURI {
```

Comment:
I replaced `ResourceURI`/`TypeURI` with just strings. With the official protobuf libs, there's not a way to override how JSON serialization works for messages, so leaving it as is would result in an extra level of nesting.